pages/docs/deployment/self-host.mdx

```diff
@@ -2,16 +2,25 @@
 description: Self-host Langfuse in your infrastructure using Docker.
 ---
 
+import { Callout } from "nextra/components";
+
 # Self-Hosting Langfuse - Open Source LLM Observability
 
 [![Docker Image](https://img.shields.io/badge/docker-langfuse-blue?logo=Docker&logoColor=white&style=flat-square)](https://github.com/langfuse/langfuse/pkgs/container/langfuse)
 
+<Callout type="info">
```

Comment:
@marcklingen But it might also be fair to announce that something new is around the corner, if we expect that the timeline is sub 4 weeks. I thought having this as an informational block at the top makes sense.

No strong preference though, i.e. happy to remove it in case you think it adds confusion.

Reply:
Understood, I'd remove it for now. It is discoverable in the menu and I would not want to confuse people who just want to get the ob done of deploying a prod version of langfuse. I think these users should not try to deploy the alpha version (not prod ready) and should not need to read the migration guide already (confusing, not polished yet and meant for early adopters)

Reply:
Agree with Marc here.

Reply:
Dropped it in https://github.com/langfuse/langfuse-docs/pull/948/commits/841a6258c9c1d425adaeb9ad44a8999c0972e06d.